torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20655,6 +20699,24 @@ def __init__(
     #
     # Elementwise Ternary Reference OpInfos
     #
+    ElementwiseBinaryPythonRefInfo(
+        "_refs.clamp_min",
+        torch_opinfo_name="clamp_min",
+        supports_nvfuser=False,
+        skips=(
+            # test error disabled since rhs non-tensor python scalar is supported
+            DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_python_ref_errors'),
```

Comment:
Convert these skips to xfails so when the issue is fixed we know to enable the test